packages/theme/src/theme/_forms.scss

```diff
@@ -82,7 +82,8 @@ form {
 		border-bottom: $form-border-bottom;
 
 		&:focus {
-			@include box-shadow($form-focus-shadow);
+			border-bottom: $form-focus-border-bottom;
+			box-shadow: none;
```

Comment:
Why do you have to fix `box-shadow: none` if none of them use it?

Reply:
fixed, reverting border-bottom styles with box-shadow as it introduced alignment issues with input fields